ENIGMAsystem/SHELL/Platforms/xlib/Makefile

```diff
@@ -1,4 +1,11 @@
 SOURCES += $(wildcard Platforms/xlib/*.cpp) $(wildcard Platforms/General/POSIX/*.cpp)
-override CXXFLAGS += $(shell pkg-config x11 --cflags)
-override CFLAGS += $(shell pkg-config x11 --cflags)
+ifeq ($(OS), Linux)
+	SOURCES += $(wildcard Platforms/xlib/Linux/*.cpp)
```

Comment:
the pkg config stuff shouldnt need to be in the if here as its exactly same for both oses